src/hooks/useCeleste.ts

```diff
@@ -70,27 +80,34 @@ function useCelesteConfigPoll(chainId) {
   return config
 }
 
-export function useCelesteSynced(chainId) {
+export function useCelesteSynced(chainId: number) {
   const config = useCelesteConfigPoll(chainId)
 
   return useMemo(() => {
     if (!config || config.terms.length === 0) {
       return [!config, false]
     }
 
-    const { currentTerm, termDuration, terms } = config
+    const { currentTerm } = config
 
     const nowS = Date.now() / 1000
-    const expectedCurrentTerm = getExpectedCurrentTermId(nowS, {
-      terms,
-      termDuration,
-    })
+    const expectedCurrentTerm = getExpectedCurrentTermId(nowS, config)
 
-    return [parseInt(expectedCurrentTerm) === parseInt(currentTerm), false]
+    if (expectedCurrentTerm !== null) {
+      return [expectedCurrentTerm === parseInt(currentTerm), false]
+    }
+
+    return null
   }, [config])
 }
 
-function getExpectedCurrentTermId(now, { terms, termDuration }) {
-  const firstTermStartTime = terms[0].startTime
-  return Math.floor((now - firstTermStartTime) / termDuration)
+function getExpectedCurrentTermId(now: number, config: ConfigType) {
+  if (config !== null) {
+    const firstTermStartTime = config?.terms[0].startTime
+    return Math.floor(
+      (now - Number(firstTermStartTime)) / Number(config?.termDuration)
```

Comment:
Appear with config !==null we dont need config? so we can remove "?" from both line. Keeping too not do harm.

Reply:
Indeed. Will update it.